src/main/java/pages/IdentifyVehiclePage.java

```diff
@@ -53,23 +54,14 @@ public boolean isSearchFieldEmpty() {
     public boolean isIncompleteRecordPopupShown() {
         System.out.println("Checking if the 'Incomplete Record' popup is shown...");
         boolean status = false;
-        boolean isException = false;
-        WebElement okButton = null;
         try {
-            okButton = findElementById(OK_ID);
+            findElementById(OK_ID);
+            findElementById(INCOMPLETE_RECORD_TITLE_ID);
+            findElementById(INCOMPLETE_RECORD_DESCRIPTION_ID);
         } catch (Exception e) {
-            isException = true;
+            status = true;
```

Comment:
Think the logic's the wrong way round on this.  It's checking for the presence of the "Incomplete record" popup.  If the three elements are present, then the status should be true - the code update returns a true if it fails to find the popup controls.